pkgs/development/python-modules/flask-wtf/default.nix

```diff
@@ -32,10 +32,14 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  disabledTests = [
+    "test_set_default_message_language"
```

Comment:
Please add a comment why this test is disabled

Reply:
> Arguably you could just pull your fix as a patch, we may forget to ever remove the test disablement.

Thanks for the review.
I removed this disabled test and use the fix as a patch in the package.